### PR TITLE
Add system.env, system.profile, system.proxy and configure proxy for repo

### PR DESCRIPTION
**Pull request by epcim**
Add system.env, system.profile and and configure proxy under system.repo

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -708,16 +770,67 @@ OpenVswitch Bridges
...
+      system:
+        env:
+          BOB_VARIABLE: Alice
+          ...
+          BOB_PATH:
+            - /srv/alice/bin
+            - /srv/bob/bin
+          ...
+          global_locales: |
+            LC_ALL=en_US
```
Comment (fpytloun) [inline README.rst]: I think this way should not be allowed for environment - you can only set key=value so stick with yaml.

File: linux/files/etc_profile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+{%- for name,value in profiles.iteritems() %}
+
+{%- if value is sequence and value is not string %}
+{{ name }}="{{ value|join(':') }}"
```
Comment (fpytloun) [inline linux/files/etc_profile]: You should always use export imho

File: linux/files/etc_profile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+{%- elif value.split('\n')|length > 1 %}
+
+# {{ name }}
+{{ value }}
+
+{%- else %}
+{{ name }}="{{ value }}"
```
Comment (fpytloun) [inline linux/files/etc_profile]: same here

File: linux/files/etc_profile [hunk 4]
```diff
@@ -0,0 +1,17 @@
+
+{%- for name,value in profiles.iteritems() %}
```
Comment (fpytloun) [inline linux/files/etc_profile]: Main purpose of profile.d is to place basically any script in there, not only for exporting env variables - this main purpose should be also main purpose of linux.system.profile

**Comment by epcim** [comment]
Jeste nemergovat. Je tam issue.

**Pull request merged**